js/about/preferences.js

```diff
@@ -576,6 +672,7 @@ class PaymentsTab extends ImmutableComponent {
                 {this.btcToCurrencyString(this.props.ledgerData.get('balance'))}
                 </span>
                 {this.walletButton}
+                {this.paymentHistoryButton}
```

Comment:
this button should not show up if there is empty payment history (ex: before wallet has been created)